management/src/clusterm/manager/api.go

```diff
@@ -72,11 +73,12 @@ func (m *Manager) apiLoop(errCh chan error) {
 	}
 }
 
-type postCallback func(tagsOrAddrs []string, sanitizedExtraVars string) error
+type postCallback func(tagsOrAddrs []string, flags ActionFlags) error
```

Comment:
@mapuri ok, then i will just make the other changes and let us take care of #99 separately